### DISCUSSION TITLE
LES with wall models

### DISCUSSION THREAD

**Discussion by MarcoArtiano** [xcompact3d/Incompact3d#108] [Q&A] [answered]
Hello,
I would like to make some LES run with wall models, I saw that in the les_models.f90 file there's an option where the divergence of the SGS stresses could be computed through the use of wall functions. I would like to learn more about that and eventually implement it by myself; is there any paper or book that can I read to learn more about it ? Moreover, computing the divergence of the SGS stresses through the wall functions is different from a wall model, where the near wall region is completely assumed to follow a rans solver or analytical wall functions ? Thank you in advance

**Comment by slaizet** [comment, marked as answer]
Hi,

The book from Sagaut is a good starting point, https://link.springer.com/book/10.1007/b137536.

You should also have a look at the PhD work of G. Deskos, he implemented some wall models in the wind farm simulator part of Xcompact3d, see [https://spiral.imperial.ac.uk/handle/10044/1/68512](https://spiral.imperial.ac.uk/handle/10044/1/68512)

All the best,

Sylvain

**Comment by MarcoArtiano** [comment]
Thank you so much for the help !
